extensions-core/namespace-lookup/src/main/java/io/druid/server/namespace/cache/OffHeapNamespaceExtractionCacheManager.java

```diff
@@ -56,15 +53,11 @@
   @Inject
   public OffHeapNamespaceExtractionCacheManager(
       Lifecycle lifecycle,
-      @Named("namespaceExtractionFunctionCache")
-      ConcurrentMap<String, Function<String, String>> fnCache,
-      @Named("namespaceReverseExtractionFunctionCache")
-      ConcurrentMap<String, Function<String, List<String>>> reverseFnCache,
       ServiceEmitter emitter,
-      final Map<Class<? extends ExtractionNamespace>, ExtractionNamespaceFunctionFactory<?>> namespaceFunctionFactoryMap
+      final Map<Class<? extends ExtractionNamespace>, ExtractionNamespaceCacheFactory<?>> namespaceFunctionFactoryMap
```

Comment:
@drcrallen i am not detecting anything, i am talking about [this] (https://github.com/druid-io/druid/blob/f2b68642b416be1199855323b87b5a176fb77f23/extensions-core/namespace-lookup/src/main/java/io/druid/server/namespace/cache/OffHeapNamespaceExtractionCacheManager.java#L117-L117)